worker-data-processing-container/start.sh

```diff
@@ -77,7 +77,7 @@ should_start_elasticsearch() {
 start_elasticsearch() {
     /opt/elasticsearchConfig/configureElasticsearch.sh
     echo "Attempting to start Elasticsearch ..."
-    /etc/init.d/elasticsearch start
+    /opt/elasticsearch/bin/elasticsearch -Des.insecure.allow.root=true -d
```

Comment:
The way this is being started it's not clear to me where the logs are going. Does this meet the logging standard? Also, I think we should look into whether that allow root property can be set in a properties file in the base image so that it doesn't have to be set in the script. If not, then we need to create a base image wrapper script.